browser/css/settings.css

```diff
@@ -0,0 +1,46 @@
+.iframe-settings-wrap {
+	position: fixed;
+	margin-inline-start: 600px;
+	margin-top: 400px;
+}
+
+.iframe-settings-content {
```

Comment:
These classes are never used in the code, could you please check and remove them?

Reply:
They are added here: https://github.com/CollaboraOnline/online/blob/df57951f27a8628e8331f7a34a71cb57832020be/browser/src/control/IFrameDialog.js#L29